src/app-interoperability/examples/valid/ApplicationDescription-001.yaml

```
@@ -0,0 +1,58 @@
+apiVersion: margo.org/v1-alpha1
+kind: application
+metadata:
+  id: com-northstartida-hello-world
+  name: Hello World
+  description: A basic hello world application
+  version: 1.0
```

@arne-broering version is a string, so quoting is needed in YAML to avoid the interpretation as a float

```suggestion
  version: "1.0"
```